spectr/src/custom_segmenter.py

```diff
@@ -0,0 +1,96 @@
+from typing import Dict, List, Any
+import pdb
+
+import torch
+import torch.nn as nn
+import pytorch_lightning as pl
+from encoder_network import EncoderNetwork
+from masked_decoder import MaskedTransformer
```

Comment:
now that we are importing spectr as a module, let's change lines 7 and 8 to:

`from spectr.src.encoder_network import EncoderNetwork`
`from spectr.src.masked_decoder import MaskedTransformer`